sentry-android-ndk/src/main/java/io/sentry/android/ndk/NdkScopeObserver.java

```diff
@@ -0,0 +1,43 @@
+package io.sentry.android.ndk;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import io.sentry.Breadcrumb;
+import io.sentry.IScopeObserver;
+import io.sentry.protocol.User;
+
+public class NdkScopeObserver implements IScopeObserver {
```

Comment:
M: I'd wrap all those methods in try/catch blocks, JNI calls are always doubtful, so at least we don't crash the App. if something goes wrong.